lib/fastlane/plugin/firebase_app_distribution/client/aab_info.rb

```diff
@@ -0,0 +1,42 @@
+class AabInfo
+  # AAB states
+  class AabState
+    UNSPECIFIED = "AAB_STATE_UNSPECIFIED"
+    PLAY_ACCOUNT_NOT_LINKED = "PLAY_ACCOUNT_NOT_LINKED"
+    NO_APP_WITH_GIVEN_BUNDLE_ID_IN_PLAY_ACCOUNT = "NO_APP_WITH_GIVEN_BUNDLE_ID_IN_PLAY_ACCOUNT"
+    APP_NOT_PUBLISHED = "APP_NOT_PUBLISHED"
+    PLAY_IAS_TERMS_NOT_ACCEPTED = "PLAY_IAS_TERMS_NOT_ACCEPTED"
+    ACTIVE = "ACTIVE"
+    UNAVAILABLE = "AAB_STATE_UNAVAILABLE"
+  end
+
+  def initialize(response)
+    @response = response || {}
+  end
+
+  def integration_state
+    @response[:integrationState]
+  end
+
+  def test_certificate
+    @response[:testCertificate]
```

Comment:
Small DRY nit - you could implement this as:
```
def test_certificate
  @response[:testCertificate] || {}
end
```

And then dispense with the ternary `test_certificate ? test_certificate[:<key>] : nil` in favor of just`test_certificate[:<key>]` in the other three methods.

Reply:
Done.